MuvrKit/MKActivationFunctions.swift

```diff
@@ -0,0 +1,55 @@
+import Foundation
+import Accelerate
+
+///
+/// Logistic function: http://en.wikipedia.org/wiki/Logistic_function
+///
+public func sigmoidActivation(inout input: [Float], offset: Int, length: Int) {
```

Comment:
Not really, since the input is a big buffer that gets reused for each layer and is therefore as big as the biggest layer needs it to be. Hence the offset and length to make sure we only process data we actually need.